js/components/dataset/quality-section.vue

```diff
@@ -0,0 +1,69 @@
+<template>
+<div>
+    <h3 class="pointer" :class="textClass" @click="toggle">
```

Comment:
I wonder how vuejs deal with that kind of double declaration of classes. Maybe we should add static one to the dynamic ones to keep all in the same place?